### PR TITLE
[stripe-v3] Add typings for paymentRequestButton.on('paymentmethod')

### PR DESCRIPTION
**Pull request by P4sca1**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://stripe.com/docs/stripe-js/reference#payment-response-object
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Comment by P4sca1** [comment]
The errors do not seem to relate to my changes. Anybody got an idea?

**Comment by slangeder** [comment]
Check the CI logs:
`Error in stripe-v3
Error: /home/travis/build/DefinitelyTyped/DefinitelyTyped/types/stripe-v3/index.d.ts:625:1
ERROR: 625:1   no-trailing-whitespace  trailing whitespace
ERROR: 627:28  expect                  TypeScript@next compile error: 
Cannot find name 'PaymentMethod'.
    at /home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:190:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/travis/build/DefinitelyTyped/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
    at <anonymous>`

File: types/stripe-v3/index.d.ts [hunk 1]
```diff
@@ -622,13 +622,18 @@ declare namespace stripe {
         interface StripeSourcePaymentResponse extends StripePaymentResponse {
             source: Source;
         }
+        
+        interface StripePaymentMethodPaymentResponse extends StripePaymentResponse {
+            paymentMethod: PaymentMethod;
```
Comment (slangeder) [inline types/stripe-v3/index.d.ts]: There is no PaymentMethod interface or class. DId you mean paymentMethod.PaymentMethod?

**Review by slangeder — CHANGES_REQUESTED** [review]

File: types/stripe-v3/index.d.ts (hunk 1 above)
Comment (P4sca1) [reply types/stripe-v3/index.d.ts]: Weird, within Webstorm this seemed to work totally fine.
I tried changing it, lets have a look.

**Review by slangeder — APPROVED** [review]

**Pull request merged**